tech/languages/dotnet/about.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: .NET
+title: .NET on Fedora
```

Comment:
Everything's `on Fedora` here. No need to remind users.

Reply:
I would suggest about.md to be `.NET Family` and dotnet.md then `.NET Core` since the first one contains mono and dotnet core, where both are of dotnet family, and the second only info about dotnet core.

Reply:
or: .NET Platforms

Reply:
`.NET Family`, if possible.

Reply:
I've replaced it with platforms. This is a term Microsoft uses when talking about mono, .NET Core, .NET Framework.

Reply:
Oh, right... any reference?